GF_glyphsets/Latin/languages/GF_Latin_Core.yaml

```diff
@@ -0,0 +1,29 @@
+language_codes:
+  - ca_Latn  # Catalan
+  - cs_Latn  # Czech
+  - cy_Latn  # Welsh
+  - da_Latn  # Danish
+  - de_Latn  # German
+  - en_Latn  # English
+  - es_Latn  # Spanish
+  - fi_Latn  # Finnish
+  - fr_Latn  # French
+  - gn_Latn  # Guarani
+  - hr_Latn  # Croatian
+  - hu_Latn  # Hungarian
+  # - is_Latn  # Icelandic
+  - it_Latn  # Italian
+  # - lt_Latn  # Lithuanian
+  # - lv_Latn  # Latvian
+  # - mt_Latn  # Maltese
+  - nb_Latn  # Norwegian Bokmål
```

Comment:
let's try to go back to this PR since Jan is back; I think here we just need to unmute these languages and we would be good to go for a first round. We can then test what would change if we lower the threshold of number of speakers. cc @yanone

Reply:
I'll go over this soon. I also want to write some documentation about the transition (in the READMEs) and define steps how to produce the new files.